Apps/phsmime/smime_consts.py

```diff
@@ -0,0 +1,33 @@
+# File: smime_consts.py
+# Copyright (c) 2020 Splunk Inc.
+#
+# SPLUNK CONFIDENTIAL - Use or disclosure of this material in whole or in part
+# without a valid written license from Splunk Inc. is PROHIBITED.
+
+# Sign email action related constants
+SMIME_SIGN_PROGRESS_MSG = "Signing email message"
+SMIME_SIGN_OK_MSG = "Email message signed successfully"
+SMIME_SIGN_ERR_MSG = "Error occurred while signing message. {err}"
+
+# Verify email action related constants
+SMIME_VERIFY_PROGRESS_MSG = "Verifying signed email message"
+SMIME_VERIFY_OK_MSG = "Signed email message verified successfully"
+SMIME_VERIFY_ERR_MSG = "Error occurred while verifying message. {err}"
+SMIME_VERIFY_ERR2_MSG = "The received message was not signed"
+
+# Encrypt email action related constants
+SMIME_ENCRYPT_PROGRESS_MSG = "Encrypting email message"
+SMIME_ENCRYPT_OK_MSG = "Email message encrypted successfully"
+SMIME_ENCRYPT_ERR_MSG = "Error occurred while encrypting message. {err}"
+
+# Decrypt email action related constants
+SMIME_DECRYPT_PROGRESS_MSG = "Decrypting encrypted email message"
+SMIME_DECRYPT_ERR_MSG = "Error occurred while decrypting message. {err}"
+SMIME_DECRYPT_ERR2_MSG = "The received message was not encrypted"
+SMIME_DECRYPT_OK_MSG = "Email message decrypted successfully"
+
+# Error message handling constants
+ERROR_CODE_MSG = "Error code unavailable"
```

Comment:
Can we change `ERROR` to `ERR` for consistency please

Reply:
Made changes as per suggestion in the latest commit: https://github.com/phantomcyber/phantom-apps/pull/365/commits/ae697c37671afeaf9378a1b9bd435848d9303993